docs/how-to/hip_runtime_api/memory_management/virtual_memory.rst

```diff
@@ -103,16 +140,40 @@ devices.
 .. code-block:: cpp
 
     hipMemAccessDesc accessDesc = {};
-    accessDesc.location.type = HIP_MEM_LOCATION_TYPE_DEVICE;
+    accessDesc.location.type = hipMemLocationTypeDevice;
     accessDesc.location.id = currentDev;
-    accessDesc.flags = HIP_MEM_ACCESS_FLAGS_PROT_READWRITE;
+    accessDesc.flags = hipMemAccessFlagsProtReadwrite;
     hipMemSetAccess(ptr, padded_size, &accessDesc, 1);
 
 At this point the memory is allocated, mapped, and ready for use. You can read
 and write to it, just like you would a C style memory allocation.
 
+.. _usage_virtual_memory:
+
+Dynamically increase allocation size
+------------------------------------
+
+The :cpp:func:`hipMemAddressReserve` function allows you to increase the amount
+of pre-allocated memory. This function accepts a parameter representing the
+requested starting address of the virtual memory. This allows you to have a
```

Comment:
```suggestion
To increase the amount of pre-allocated memory use :cpp:func:`hipMemAddressReserve`;
which accepts the starting address, and the size of the reservation in bytes.  This allows you to have a
```
Unclear sentences

Reply:
We try to avoid `;`. That's why I made slight changes to your suggestion. Please check the outcome.

Reply:
Thanks, looks good